poppy/poppy_core.py

```diff
@@ -2104,15 +2136,20 @@ def getPhasor(self,wave):
         else:
             return self.phasor
 
+    def getPhasor(self,wave):
+        warnings.warn("getPhasor is deprecated; use get_phasor instead", DeprecationWarning)
+        return self.get_phasor(wave)
+
+
     def display(self, nrows=1, row=1, what='intensity', crosshairs=True, ax=None, colorbar=True,
```

Comment:
That's a little off-topic of the pull request, just reminded it had bugged me in the past when I was skimming. PR seems good in five minute skim.